addons/xeh/fnc_initDisplay3DEN.sqf

```diff
@@ -8,14 +8,5 @@ add3DENEventHandler ["OnMissionPreviewEnd", {[] call CBA_fnc_preInit}];
 
 // switching terrains in 3den will reset missionNamespace
 add3DENEventHandler ["OnTerrainNew", {[] call CBA_fnc_preInit}];
-
-private _fnc_preInitMissionConfig = {
-    {
-        if (_x select 0 == "" && {_x select 1 == "preInit"}) then {
-            [] call (_x select 2);
-        };
-    } forEach (missionConfigFile call CBA_fnc_compileEventHandlers);
-};
-
-add3DENEventHandler ["OnMissionNew", _fnc_preInitMissionConfig];
-add3DENEventHandler ["OnMissionLoad", _fnc_preInitMissionConfig];
+add3DENEventHandler ["OnMissionNew", {[] call CBA_fnc_preInit}];
```

Comment:
Going from one terrain to another
```
"OnMissionNew"
XEH: missionNamespace processed [false]
[83008,343.676,0,"XEH: PreInit started. v3.3.1.170504"]
[83008,344.812,0,"XEH: PreInit finished."]
"OnTerrainNew"
XEH: missionNamespace processed [false]
83008,346.107,0,"XEH: PreInit started. v3.3.1.170504"]
```
both events happen on the same frame, but missionNamespace is wiped between them
`OnTerrainNew` seems to be last, so if we don't add the event, nothing will be defined